### ISSUE TITLE
Is it possible to deIs it possible to delete predictors?

### ISSUE THREAD

**Issue by tawichi** [tomas-gajarsky/facetorch#48]
I don't need the deep fake function for my application, so I want to turn it off to reduce execution time. Is this possible?

**Comment by tomas-gajarsky** [comment]
It is possible to disable predictors by removing them from the configuration.

- If you are using the main `config.yaml` file that joins all the partial configs, you can just remove [the line with the deepfake predictor](https://github.com/tomas-gajarsky/facetorch/blob/main/conf/config.yaml#L10).
- If you are using the `merged.config.yaml` file, then you need to remove [the entire configuration of the deepfake predictor](https://github.com/tomas-gajarsky/facetorch/blob/main/conf/merged/merged.config.yaml#L306C5-L348).

**Comment by tawichi** [comment]
Thank you, I was able to turn off the deep fake feature. I also tried to delete the align predictor but was unable to do so.
Is it not possible to turn off the align function?

**Comment by tawichi** [comment]
You are right, I deleted both predictor and utilizer and it worked. Thank you very much!
